frame/staking/src/benchmarking.rs

```diff
@@ -493,13 +503,13 @@ benchmarks! {
 
 	// same as submit_solution_initial but we place a very weak solution on chian first.
 	submit_solution_better {
-		// number of validator intent
-		let v in 1000 .. 2000;
-		// number of nominator intent
-		let n in 1000 .. 2000;
+		// number of validator intention.
+		let v in 200 .. 400;
```

Comment:
where do these ranges come from? Seems to me something we may want to expose to the user to configure for their own personal benchmarks right?

Reply:
Nowhere special, they should just be large enough so the benchmark formula can work properly. Here I reduced the numbers because we get the same formula, while the execution times of these benchmarks was annoyingly high, so I reduced them. I will keep this open but I don't see any reason to expose these.

Reply:
For example I doubt if increasing the validator count in the benchmarks would make any difference here. The formula is the same in any case.